plugins/vue-example/frontend/public/javascripts/countly.models.js

```diff
@@ -0,0 +1,240 @@
+/*global $, countlyCommon, _, countlyVue */
+
+(function(countlyVueExample) {
```

Comment:
Codacy found an issue: [Move the invocation into the parens that contain the function.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=6339769)